main.tf

```diff
@@ -0,0 +1,105 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-1"
+}
```

Comment:
It's a good idea to move this file into a separate file. You can call it `providers.tf`